<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindSystem</class>
 <widget class="QDialog" name="WindSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#WindSystem{
	background-color:rgb(178,178,178,255);
}</string>
  </property>
  <widget class="QFrame" name="bottom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#bottom
{
	background-color:#ffffff;
 
	border:none;
	border-radius:25px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>新风强度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="ButtonBox">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>330</y>
      <width>231</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#ButtonBox{
border:none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="StopMode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/new/Curtain/Curtain/Stop_OFF.png</normaloff>
       <normalon>:/new/Curtain/Curtain/Stop_ON.png</normalon>:/new/Curtain/Curtain/Stop_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="MidMode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/new/Wind/Wind/Mid_OFF.png</normaloff>
       <normalon>:/new/Wind/Wind/Mid_ON.png</normalon>
       <disabledoff>:/new/Wind/Wind/Mid_OFF.png</disabledoff>:/new/Wind/Wind/Mid_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="HighMode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/new/Wind/Wind/High_OFF.png</normaloff>
       <normalon>:/new/Wind/Wind/High_ON.png</normalon>:/new/Wind/Wind/High_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="MinMode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/new/Wind/Wind/Low_OFF.png</normaloff>
       <normalon>:/new/Wind/Wind/Low_ON.png</normalon>
       <disabledoff>:/new/Wind/Wind/Mid_OFF.png</disabledoff>:/new/Wind/Wind/Low_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;
border-radius:25px;
background-color: white;
 border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../other/main.qrc">
      <normaloff>:/new/Main/Air/Back.png</normaloff>:/new/Main/Air/Back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Pirture">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../other/main.qrc"/>
 </resources>
 <connections/>
</ui>
